diagnostic_feedback/templates/underscore/category.html

```diff
@@ -0,0 +1,13 @@
+<div class="category">
+    <input type="hidden" id="category[id][<%- order %>]" name="category[id][<%- order %>]" class="skip-validation" value="<%- id %>" />
+    <input placeholder="Name" type="text" id="category[name][<%- order %>]" name="category[name][<%- order %>]"  class="required" value="<%- name %>" />
+    <input placeholder="Image" type="text" id="category[image][<%- order %>]" name="category[image][<%- order %>]"  value="<%- image %>" />
+    <a href="#" data-tooltip="Delete" class="delete-category delete-button action-button">
+        <i class="icon fa fa-trash-o"></i>
+        <span class="sr">Delete</span>
+    </a>
+    <input placeholder="Internal Description" class="internal-description" type="text"
```

Comment:
Would be good if the `placeholder` had a bit more explanation, perhaps `Internal Description (Notes for author; not shown to students)` ?